src/gt4py/backend/gtc_common.py

```diff
@@ -261,14 +261,14 @@ def generate_computation(self) -> Dict[str, Union[str, Dict]]:
         return {dir_name: src_files["computation"]}
 
     def generate_bindings(
-        self, language_name: str, *, ir: Any = None
+        self, language_name: str, *, stencil_ir: gtir.Stencil = None
     ) -> Dict[str, Union[str, Dict]]:
-        if not ir:
-            ir = self.builder.gtir
+        if not stencil_ir:
+            stencil_ir = self.builder.gtir
         if language_name != "python":
             return super().generate_bindings(language_name)
         dir_name = f"{self.builder.options.name}_src"
-        src_files = self.make_extension_sources(stencil_ir=ir)
```

Comment:
Is this an unrelated change? (Not against it...)

Reply:
The previous PR was supposed to rename all the `ir` to `stencil_ir`, but I must have forgotten this one :).